rules_poetry/poetry.bzl

```diff
@@ -79,8 +94,31 @@ def _mapping(repository_ctx):
     return {
         "dependencies": dependencies,
         "groups": groups,
+        "pyproject": pyproject
     }
 
+def extract_markers(repository_ctx, resolved_markers, dep, markers):
+    python_interpreter = _resolve_python_interpreter(repository_ctx)
+
+    if str(type(markers)) == 'list':
+        if dep not in resolved_markers:
+            resolved_markers[dep] = {}
+        for marker in markers:
+            for platform in SUPPORTED_PLATFORMS:
+                system, machine = platform.split('!')
+                test_string = marker['markers'].replace('platform_machine', "'" + machine + "'")
+                test_string = test_string.replace('sys_platform', "'" + system + "'")
+                cmd = [
+                    python_interpreter,
```

Comment:
I just want to test a condtional like `'s390x' == 's390x'` or `'s390x' != 's390x'`, I originally hoped that starlark could `eval` but its not the case so I need to build the code string and pass it to something to eval it. In this the python interpretor was available and is cross platform in nature.